packages/terra-form-radio/package.json

```diff
@@ -28,6 +28,7 @@
     "classnames": "^2.2.5",
     "lodash.uniqueid": "^4.0.1",
     "prop-types": "^15.5.8",
+    "raf-polyfill": "^1.0.0",
```

Comment:
Why was this added?

Reply:
No idea. Removed: https://github.com/cerner/terra-core/pull/2314/commits/6374930ee51190515c85b4726ae6514a15f42c67#diff-afd97ad7b73244896cf178257a620ee2L31